content/tutorials/signing-transactions-with-webauthn/30.building-the-frontend.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Building the Frontend
+description: Build the frontend for your app.
+
+---
+
+## Frontend Setup
+
+We will use Next.js to build the frontend.
+
+Move out of your `contract` folder and back into the `zksync-webauthn` folder.
+Make a new Next.js project for the frontend using the configuration below:
+
+```shell
+npx create-next-app@latest
+```
+
+```txt
+✔ What is your project named? … frontend
+✔ Would you like to use TypeScript? Yes
+✔ Would you like to use ESLint? Yes
+✔ Would you like to use Tailwind CSS? No
+✔ Would you like to use `src/` directory? Yes
+✔ Would you like to use App Router? (recommended) No
+✔ Would you like to customize the default import alias (@/*)? No
+```
+
+Move into your projects and install the dependencies below:
+
+```shell
+cd frontend
+npm install @simplewebauthn/browser@10.0.0 @simplewebauthn/server@10.0.1 cbor@9.0.2 ethers@5.7.2 zksync-ethers@5.1.0
```

Comment:
any particular reason for zksync-ethers v5 usage here?

Reply:
According to the Clave team there are some things they still needed to work out using ethers v6 for this implementation.